examples/resources/powerscale_smartpool_settings/resource.tf

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright (c) 2023 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Mozilla Public License Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://mozilla.org/MPL/2.0/
+
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+# Available actions: Create, Update, Delete
+# After `terraform apply` of this example file it will create a state file that presents the SmartPools Settings on the PowerScale.
+# For more information, Please check the terraform state file.
+
+resource "powerscale_smartpool_settings" "settings" {
```

Comment:
don't need import.sh ?

Reply:
when apply tf without any argument, the smartpool settings will be loaded from PowerScale to Terraform, just like import
```
resource "powerscale_smartpool_settings" "settings" {
}
```

Reply:
@baoy1  yes, the create will do import thing, but thinking import also is one part of feature. Not sure import is not needed.

Reply:
Added import in case customer is not aware that create without arguments behaves the same.

Reply:
BTW, Import requires ID is defined in resource, though that is not necessary for SmartPool settings.